--- rpmlint-1.7/Makefile~	2015-07-15 15:10:27.000000000 +0300
+++ rpmlint-1.7/Makefile	2015-07-15 15:12:32.665044829 +0300
@@ -10,6 +10,7 @@
 LIBDIR=/usr/share/rpmlint
 ETCDIR=/etc
 MANDIR=/usr/share/man
+bash_compdir=$(shell pkg-config --variable=completionsdir bash-completion 2>/dev/null)
 
 FILES = rpmlint *.py INSTALL README README.devel COPYING tools/*.py \
 	Makefile config rpmdiff rpmdiff.1 rpmlint.bash-completion rpmlint.1 \
@@ -48,14 +48,13 @@
 		chmod +x $(DESTDIR)$(BINDIR)/$$file ; \
 	done
 	cp -p config $(DESTDIR)$(ETCDIR)/$(PACKAGE)
-	compdir=`pkg-config --variable=completionsdir bash-completion 2>/dev/null` ; \
-	if [ "x$$compdir" = "x" ] ; then \
+	if [ "x$(bash_compdir)" = "x" ] ; then \
 		mkdir -p $(DESTDIR)$(ETCDIR)/bash_completion.d ; \
 		cp -p rpmlint.bash-completion $(DESTDIR)$(ETCDIR)/bash_completion.d/rpmlint ; \
 	else \
-		mkdir -p $(DESTDIR)$$compdir ; \
-		cp -p rpmlint.bash-completion $(DESTDIR)$$compdir/rpmlint ; \
-		ln -sf rpmlint $(DESTDIR)$$compdir/rpmdiff ; \
+		mkdir -p $(DESTDIR)$(bash_compdir) ; \
+		cp -p rpmlint.bash-completion $(DESTDIR)$(bash_compdir)/rpmlint ; \
+		ln -sf rpmlint $(DESTDIR)$(bash_compdir)/rpmdiff ; \
 	fi
 	cp -p rpmdiff.1 rpmlint.1 $(DESTDIR)$(MANDIR)/man1
 
