--- rpmlint-1.0/DocFilesCheck.py~	2010-05-19 17:54:28.000000000 +0300
+++ rpmlint-1.0/DocFilesCheck.py	2011-01-09 18:57:11.207990792 +0200
@@ -40,7 +40,7 @@
 
         for dep in pkg.header.dsFromHeader():
             # skip deps which were found by find-requires
-            if dep.Flags() & rpm.RPMSENSE_FIND_REQUIRES != 0:
+            if dep.Flags(): # & rpm.RPMSENSE_FIND_REQUIRES != 0:
                 continue
             core_reqs[dep.N()] = []
 
--- rpmlint-1.0/Pkg.py~	2011-01-09 18:52:55.477166839 +0200
+++ rpmlint-1.0/Pkg.py	2011-01-09 18:55:25.437670292 +0200
@@ -42,11 +42,7 @@
 
 # 64: RPMSENSE_PREREQ is 0 with recent rpm versions, we want 64 here in order
 # to do the right thing with packages built with older rpm versions
-PREREQ_FLAG = (rpm.RPMSENSE_PREREQ or 64) | \
-              rpm.RPMSENSE_SCRIPT_PRE | \
-              rpm.RPMSENSE_SCRIPT_POST | \
-              rpm.RPMSENSE_SCRIPT_PREUN | \
-              rpm.RPMSENSE_SCRIPT_POSTUN
+PREREQ_FLAG = (rpm.RPMSENSE_PREREQ or 64)
 
 var_regex = re.compile('^(.*)\${?(\w+)}?(.*)$')
 
