--- rpmlint-1.4/Pkg.py~	2012-09-25 22:39:39.673226624 +0200
+++ rpmlint-1.4/Pkg.py	2012-09-25 22:40:12.223225476 +0200
@@ -434,7 +434,7 @@
             # Create a package object from the file name
             ts = rpm.TransactionSet()
             # Don't check signatures here...
-            ts.setVSFlags(rpm._RPMVSF_NOSIGNATURES)
+            ts.setVSFlags(rpm.RPMVSF_NODSAHEADER)
             fd = os.open(filename, os.O_RDONLY)
             try:
                 self.header = ts.hdrFromFdno(fd)
