--- rpmlint-1.0/config	2011-01-09 19:00:18.531839097 +0200
+++ rpmlint-1.0/config	2011-01-09 21:14:41.284427355 +0200
@@ -27,7 +27,7 @@
 # When checking that various files that should be compressed are
 # indeed compressed, look for this filename extension (no dot here).
 # Type: string, default: "bz2"
-#setOption("CompressExtension", "bz2")
+setOption("CompressExtension", "gz")
 
 # Exception list for dangling symlink checks.  The first in each pair
 # is a regexp, and the second the package in which the target of the
@@ -156,7 +156,7 @@
 
 # Whether %changelog entries should contain a version.
 # Type: boolean, default: True
-#setOption("UseVersionInChangelog", True)
+setOption("UseVersionInChangelog", False)
 
 # Architecture dependent paths in which packages are allowed to install files
 # even if they are all non-binary.
@@ -201,0 +202,3 @@
+
+# In PLD Linux, macros are not expanded in %changelog
+addFilter("W: macro-in-%changelog")
