--- rpmlint-1.7/Pkg.py~	2015-07-08 16:38:24.000000000 +0300
+++ rpmlint-1.7/Pkg.py	2015-07-15 15:10:07.954186622 +0300
@@ -197,7 +197,7 @@
     the rpm package (if installed) if no filename is given"""
     groups = []
     if not filename:
-        with InstalledPkg("rpm") as p:
+        with InstalledPkg("rpmlint") as p:
             groupsfiles = [x for x in p.files() if x.endswith('/GROUPS')]
             if groupsfiles:
                 filename = groupsfiles[0]
