--- rpmlint-1.0/TagsCheck.py~	2010-06-23 21:34:28.000000000 +0300
+++ rpmlint-1.0/TagsCheck.py	2011-01-09 18:36:31.642820895 +0200
@@ -34,6 +34,8 @@
     enchant = None
 del _use_enchant
 
+# See PLD License list at
+# http://cvs.pld-linux.org/cgi-bin/cvsweb.cgi/PLD-doc/license-tags.txt
 DEFAULT_VALID_LICENSES = (
     # OSI approved licenses, http://www.opensource.org/licenses/ (unversioned,
     # trailing "license" dropped based on fuzzy logic, and in well-known cases,
