--- rpmlint-1.0/config~	2010-03-02 19:39:00.000000000 +0200
+++ rpmlint-1.0/config	2011-01-09 18:33:02.713646528 +0200
@@ -156,7 +156,7 @@
 
 # Whether %changelog entries should contain a version.
 # Type: boolean, default: True
-#setOption("UseVersionInChangelog", True)
+setOption("UseVersionInChangelog", False)
 
 # Architecture dependent paths in which packages are allowed to install files
 # even if they are all non-binary.
